Update Docker image tags to use GHCR repo

tillprochaska opened the pull request:
Right now, UI tests build the Docker image twice because the tags do not match, causing CI runs to take 2-3 mins longer than necessary.

tillprochaska commented on contrib/keycloak/docker-compose.dev-keycloak.yml:
```diff
@@ -16,7 +16,7 @@ services:
     build:
       context: services/elasticsearch
-    image: alephdata/aleph-elasticsearch:${ALEPH_TAG:-latest}
+    image: ghcr.io/alephdata/aleph-elasticsearch:${ALEPH_TAG:-latest}
```
I think we might need to pin this to a commit hash tag (because we don’t have versioned releases for this image)?

Rosencrantz replied:
Yeah, I think that we should perhaps get rid of latest and pin to a version. Might even be something that we can automate in time (we don't need to do that now though)

tillprochaska commented on helm/charts/aleph/values.yaml:
Decided to update the values here too. This should be the only file that specifies the repos, all templates just reference the values. But I’m not familiar with Helm, so maybe I’m missing something!

stchris reviewed (approved):
Overall this looks good to me, thanks for this change! I think the only comment I have is that we should update this for the `3.14.x` releases and not just for `develop` and future releases!

stchris commented:
> So we should merge this into main and create a new 3.14.x release with these changes?

I think this change should go into `develop` and then we should ship it with `3.14.1` (for which there is a release branch available).

Github: comment threads when?

Rosencrantz commented on contrib/aleph-traefik-minio-keycloak/docker-compose.yml:
```diff
@@ -65,7 +65,7 @@ services:
 
   worker:
-    image: alephdata/aleph:${ALEPH_TAG:-3.12.2}
+    image: ghcr.io/alephdata/aleph:${ALEPH_TAG:-3.12.2}
```
Whilst we are here can we update the tags to be 3.14.0?

tillprochaska replied:
Right! In that case I’d also add the file to the bump2version config, ok?

I was just thinking that maybe it’s a good idea for these example configuration to be actually a "frozen" snapshot, i.e. it worked with 3.12.2 but we don’t really maintain or test these -- so it might be better to not update it at all (which includes reverting the new repo, because 3.12.2 is still available on Docker Hub).

tillprochaska commented:
@Rosencrantz @stchris As discussed earlier, I have made the following changes:

* Updated all references to outdated Aleph versions (3.12.2) to the latest Aleph version 3.14.0.
* Adjusted the bump2version configuration to update the Aleph version in Compose files in the `contrib/` directory.
* Fixed the image tag of the ES image in the Keycloak example.

stchris reviewed (approved):
LGTM!

tillprochaska commented:
Closing as all the changes are in main already, I think
